cobalt/dom/html_media_element.cc

```diff
@@ -1557,14 +1555,14 @@ void HTMLMediaElement::TimeChanged(bool eos_played) {
   // already posted one at the current movie time.
   ScheduleTimeupdateEvent(false);
 
-  float now = current_time(NULL);
-  float dur = duration();
+  const double now = current_time(NULL);
+  const double dur = duration();
 
   // When the current playback position reaches the end of the media resource
   // when the direction of playback is forwards, then the user agent must follow
   // these steps:
   eos_played |=
-      !std::isnan(dur) && (0.0f != dur) && now >= dur && playback_rate_ > 0;
+      !std::isnan(dur) && (0.0 != dur) && now >= dur && playback_rate_ > 0.f;
```

Comment:
should this have a 0.0?

Reply:
I believe this row is now good with the data types.
"dur" is now double and "0.0" is double.
"playback_rate_" is float and "0.f" is float.

PS. I don't think it would be any memory concerns regarding this change of float to double. It is no arrays with these variables. Also in several places it are already 64 bit integer instead, ie same size as double.